docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,197 @@
+
+[float]
+==== Auto-apply your changes in Lens
+
+By default, *Lens* has always automatically applied the changes you make to visualizations.
+Now you can control when *Lens* applies your changes and generates suggestions with
+the new ability to turn on and off auto-apply. To start manually applying your changes,
+click *Settings* in the toolbar, then deselect *Auto-apply visualization changes*.
+For more information, check {kibana-ref}/lens.html#create-the-visualization-panel[Create visualizations].
+
+[role="screenshot"]
+image::images/highlights-lens-auto-apply.png[Auto-apply changes in Lens]
+
+[float]
+==== Add annotations to Lens visualizations (technical preview)
+
+To call out significant changes and trends in your data, you can now add annotations
+to *Lens* visualizations in tech preview. Customize the appearance of annotations with text,
+icons, colors, and more. For more information, check {kibana-ref}/lens.html#add-annotations[Add annotations].
+
+[role="screenshot"]
+image::images/highlights-lens-annotations.png[Annotations in Lens visualzations]
+
+[float]
+==== Open TSVB visualizations in Lens
+
+Open and edit your Time Series *TSVB* visualizations in *Lens*,
+the drag-and-drop visualization editor that provides you with additional
+visualization types, reference lines, and more.
+For more information, check {kibana-ref}/tsvb.html#edit-visualizations-in-lens[Edit visualizations in Lens].
+
+[role="screenshot"]
+image::images/highlights-lens-tsvb.gif[TSVB visualization in Lens]
+
+[float]
+==== Explore Lens visualization data in Discover
+
+For further exploration of your data, you can now open *Lens* visualization data
+in *Discover*. With *Discover* you can view the documents in the fields you used
+to create the visualization, then navigate back to *Lens* to finish creating your
+visualization. For more information,
+check {kibana-ref}/lens.html#explore-lens-data-in-discover[Explore the data in Discover].
+
+[role="screenshot"]
+image::images/highlights-lens-discover.gif[Seamlessly move from Lens to Discover to further explore your data]
+
+[float]
+==== Enhancements to visualization editors
+
+{kib} offers even more ways to create visualizations:
+
+* **Metric visualization.** Use the new Metric visualization type in **Lens**
+to show the latest or earliest data, or the last value.
+
+* **Group by multiple fields in TSVB.** When you select **Terms** from the **Group by**
+dropdown in **TSVB**, you can calculate the top values for multiple fields.
+
+* **Customize the legend size.** In all visualization editors,
+you can now increase or decrease the visibility of your legend by changing the size.
+
+[float]
+=== Maps
+
+[float]
+==== Display clusters as hexagons
+
+In *Maps*, you can now display your clusters as hexagons
+instead of rectangles. This feature adds another beautiful, and more precise way, to
+visualize your geo_point fields. Hexagon aggregations require
+a basic license on Cloud and a platinum license on prem.
```

Comment:
```suggestion
a standard subscription on Cloud and a platinum license on prem.
```

I'm not 100% sure that standard is correct, it might be gold or platinum.

Reply:
I'm checking with Cloud PM to confirm.

Reply:
Fly-by comment, but this is likely the kind of detail that doesn't belong in the docs anyway.